packages/scoped-elements/package.json

```diff
@@ -43,7 +43,7 @@
   "dependencies": {
     "@lit/reactive-element": "^1.0.0-rc.1",
     "@open-wc/dedupe-mixin": "^1.3.0",
-    "@webcomponents/scoped-custom-element-registry": "0.0.1"
+    "@webcomponents/scoped-custom-element-registry": "0.0.2"
   },
   "types": "types/index.d.ts",
   "sideEffects": [
```

Comment:
The sideEffects array can be removed now I think

Reply:
That's right! thanks!